src/lib/datepicker/datepicker-input.ts

```diff
@@ -79,6 +86,8 @@ export class MatDatepickerInputEvent<D> {
     MAT_DATEPICKER_VALIDATORS,
     {provide: MAT_INPUT_VALUE_ACCESSOR, useExisting: MatDatepickerInput},
   ],
+  inputs: ['value', 'min', 'max', 'disabled', 'cdkDatepickerFilter', 'cdkDatepicker'],
```

Comment:
I think we have to rename `'cdkDatepickerFilter'` and `'cdkDatepicker'`

Reply:
Already removed them :)